app/src/main/java/com/macbitsgoa/events/aboutmac/AboutMacActivity.java

```diff
@@ -16,10 +16,10 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_about_mac);
 
-        ViewPager aboutMacViewPager = (ViewPager) findViewById(R.id.about_mac_viewpager);
+        ViewPager aboutMacViewPager = findViewById(R.id.about_mac_viewpager);
```

Comment:
why these changes are required for fest merch?

Reply:
this was redundant so i removed it

Reply:
remove all changes not required for solving this issue. 
Also I've rebased your repo master to origin master. You should pull it to local. (pull --hard if no local changes have done, else pull --rebase)

Reply:
also, rebase to latest origin master

Reply:
git is showing that my branch is up to date with origin master

Reply:
should i rebase my branch to upstream master